runtime/doc/lua.txt

```diff
@@ -3321,19 +3321,18 @@ vim.lpeg.setmaxstack({max})                           *vim.lpeg.setmaxstack()*
       • {max}  (`integer`)
 
 vim.lpeg.type({value})                                       *vim.lpeg.type()*
-    Returns the string `"pattern"` if the given value is a pattern, otherwise
-    `nil`.
+    Returns the string 'pattern `if the given value is a pattern, otherwise` nil`.
 
     Return: ~
-        (`"pattern"?`)
+        ('pattern ?`)
```

Comment:
I think this is meant to be the literal string `"pattern"`, not a type `pattern`.

Reply:
Hmmm weird. This is indeed a literal in the metafile:
https://github.com/neovim/neovim/blob/d3d10ab25cc440a3f14dcb184d91ef5dc2526c8e/runtime/lua/vim/_meta/lpeg.lua#L91

Reply:
The docgen probably doesn't understand `'literal'` strings but only does `"literal"` strings... 😂

Reply:
AHHHHHHH

Reply:
See this: https://github.com/neovim/neovim/pull/26989#discussion_r1450986576